backend-worker/src/main/kotlin/org/rucca/cheese/auth/AuthorizationService.kt

```
@@ -0,0 +1,171 @@
+
+        val payload: TokenPayload
+        try {
+            payload =
+                objectMapper.readValue(
+                    verifier.verify(tokenWithoutBearer).getClaim("payload")?.toString()
+                        ?: throw InvalidTokenError(),
+                    TokenPayload::class.java,
+                )
+        } catch (e: TokenExpiredException) {
+            throw TokenExpiredError()
+        } catch (e: JWTVerificationException) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)